package.json

```diff
@@ -1674,20 +1674,29 @@
     "taskDefinitions": [
       {
         "type": "R",
-        "required": ["command", "args"],
+        "required": ["code"],
         "properties": {
-          "command": {
-            "type": "string",
-            "default": "Rscript",
-            "description": "The command to be executed."
+          "code": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "default": [
+              "x <- 'Hello, World!'",
+              "print(x)"
+            ],
+            "description": "R code to be executed"
           },
-          "args": {
+          "options": {
```

Comment:
There are multiple terms that might look confusing: args, code args, etc.

Let's use their official terms:

```
$ R --help
Usage: R [options] [< infile] [> outfile]
```